devel/fortio/periodic.go

```diff
@@ -224,19 +221,17 @@ func runOne(id int, funcTimes *Histogram, sleepTimes *Histogram, numCalls int64,
 			targetElapsedInSec := (float64(i) + float64(i)/float64(numCalls-1)) / perThreadQPS
 			targetElapsedDuration := time.Duration(int64(targetElapsedInSec * 1e9))
 			sleepDuration := targetElapsedDuration - elapsed
-			if verbosity > 3 {
-				log.Printf("%s target next dur %v - sleep %v", tIDStr, targetElapsedDuration, sleepDuration)
-			}
+			Debugf("%s target next dur %v - sleep %v", tIDStr, targetElapsedDuration, sleepDuration)
 			sleepTimes.Record(sleepDuration.Seconds())
 			time.Sleep(sleepDuration)
 		}
 	}
 	elapsed := time.Since(start)
 	actualQPS := float64(i) / elapsed.Seconds()
-	log.Printf("%s ended after %v : %d calls. qps=%g", tIDStr, elapsed, i, actualQPS)
-	if (numCalls > 0) && (verbosity > 0) {
+	Infof("%s ended after %v : %d calls. qps=%g", tIDStr, elapsed, i, actualQPS)
+	if (numCalls > 0) && Log(Verbose) {
 		funcTimes.Log(tIDStr+" Function duration", 99)
-		if verbosity > 2 {
+		if Log(Debug) {
```

Comment:
in some intermediate version I had completely deleted the shortcuts as Log(Verbose) is as short and clear (and more generic) as LogVerbose() until I got to the main packages where there one need to use fortio.Verbose, so I put them back but only for outside